### PR TITLE
fixed bug where the isIntrinsicSize error would throw an error (#188)

### PR DESCRIPTION
**Pull request by peeke**

### CODE REVIEW

File: library/stylelint-plugins/machinery/ast.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const createValueParser = require('postcss-values-parser')
+const { parse: valuesParser } = require('postcss-values-parser')
```
Comment (EECOLOR) [inline library/stylelint-plugins/machinery/ast.js]:
```suggestion
const { parse: parseValue } = require('postcss-values-parser')
```

File: library/stylelint-plugins/machinery/ast.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ function isPseudoElement({ type, value }) {
   return type === 'pseudo' && value.startsWith('::')
 }
 
-function parseValue(value) { return createValueParser(value).parse() }
+function parseValue(value) { return valuesParser(value) }
```
Comment (EECOLOR) [inline library/stylelint-plugins/machinery/ast.js]:
```suggestion
```

**Comment by peeke** [comment]
> Did you run the stylelint tests? They may reveal other changes that do not produce an error.

I did not, will do so!

**Comment by peeke** [comment]
No test failures :+1:
Will release & merge in a few minutes

**Pull request merged**